### PR TITLE
Added DI link

### LINKED ISSUE #8265: Question about Dependency Injection
Hi,
I'm learning about dependency injection and I understood the WebContext injection. But in this line:

```
using (var context = new MovieContext(serviceProvider.GetRequiredService<DbContextOptions<MovieContext>>()))
```
It seems like the we're requesting a `DbContextOptions` service, but that was never registered. Or by registering the `DbContext` it automatically adds this service? Because it is a parameter for the constructor perhaps? I'm guessing here.

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a111b14d-2c56-f0a9-d318-f99ae6bb016e
* Version Independent ID: 91430704-1e14-9b10-9fd1-d5a19cc5d380
* Content: [Work with SQLite in an ASP.NET Core Razor Pages app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages-vsc/sql?view=aspnetcore-2.1)
* Content Source: [aspnetcore/tutorials/razor-pages-vsc/sql.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/razor-pages-vsc/sql.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by daveabrock**
Fixes #8265

### CODE REVIEW

File: aspnetcore/includes/RP/sql.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The `MovieContext` object handles the task of connecting to the database and mapping `Movie` objects to database records. The database context is registered with the [Dependency Injection](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method in the *Startup.cs* file:
+The `MovieContext` object handles the task of connecting to the database and mapping `Movie` objects to database records. The database context is registered with the [Dependency Injection (DI)](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method in the *Startup.cs* file:
 
 [!code-csharp[](code/Startup.cs?name=snippet2&highlight=6-8)]
 
+See [Using DbContext with Dependency Injection](https://docs.microsoft.com/en-us/ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection) for details on using `DbContext` with DI.
```
Comment (scottaddie) [inline aspnetcore/includes/RP/sql.md]: - Use the following relative link instead: /ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection
- Using DbContext with Dependency Injection --> Using DbContext with DI
- Rephrase this sentence to "For more information on using `DbContext` with DI, see ...". This is the structure we recently decided to adopt. It's the sentence structure recommended by an internal tool we use.

**Comment by daveabrock** [comment]
@scottaddie Would this work? I'm editing from the UI and don't know if this will resolve...

`xref:ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection`

**Comment by scottaddie** [comment]
@daveabrock Great question. The answer is no. An "xref"-style link won't work in this case because the doc you're linking to lives outside of the aspnet/Docs repo. The link syntax should be:

```
[Using DbContext with DI](/ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection)
```

**Comment by scottaddie** [comment]
@daveabrock Thank you for your contribution and patience.

**Pull request merged**